index.html

```diff
@@ -5007,13 +4980,254 @@ <h3 id="looping-AudioBufferSourceNode">
             times (in seconds) must be converted to the appropriate
             sample-frame positions in the buffer according to this sample-rate.
           </p>
+        </section>
+        <section>
+          <h3 id="playback-AudioBufferSourceNode">
+            Playback of AudioBuffer Contents
+          </h3>
+          <p>
+            This section specifies how the playback of the contents of the
+            buffer is to be handled, accounting for the fact that playback is
+            influenced by the following factors working in combination, which
+            can vary dynamically during playback:
+          </p>
+          <ul>
+            <li>A starting offset, which can be expressed in terms of fractions
```

Comment:
The starting offset in the algorithm is not expressed in fractions of a sample.  All time quantities are just pure time quantities in seconds.